crates/dbsp/src/storage/file/writer.rs

```diff
@@ -1098,13 +1098,18 @@ impl Writer {
     ) -> Result<Self, StorageError> {
         assert_eq!(factories.len(), n_columns);
 
-        let bloom_filter = BloomFilter::with_false_pos(BLOOM_FILTER_FALSE_POSITIVE_RATE)
-            .seed(&BLOOM_FILTER_SEED)
-            .expected_items({
-                // `.max(64)` works around a fastbloom bug that hangs when the
-                // expected number of items is zero (see
-                // https://github.com/tomtomwombat/fastbloom/issues/17).
-                estimated_keys.max(64)
+        let bloom_false_positive_rate =
+            Runtime::with_dev_tweaks(|dev_tweaks| dev_tweaks.bloom_false_positive_rate);
+        let bloom_filter = (bloom_false_positive_rate > 0.0 && bloom_false_positive_rate < 1.0)
```

Comment:
It may be a good idea to log once, on pipeline startup, the effective bloom filter rate.